pkg/machine/ignition.go

```diff
@@ -89,6 +90,25 @@ ExecStart=/bin/sh -c '/usr/bin/echo Ready >/dev/%s'
 [Install]
 RequiredBy=multi-user.target
 `
+	deMoby := `[Unit]
+Description=Remove moby-engine, alias podman
+# Run once for the machine
+After=systemd-machine-id-commit.service
+Before=zincati.service
+ConditionPathExists=!/var/lib/%N.stamp
+
+[Service]
+Type=oneshot
+RemainAfterExit=yes
+ExecStart=/usr/bin/rpm-ostree override remove moby-engine
+ExecStart=/bin/ln -fs /usr/bin/podman /usr/local/bin/docker
+ExecStart=/bin/ln -fs /run/podman/podman.sock /run/docker.sock
```

Comment:
```suggestion
```

This isn't going to work because `/run/` starts from scrtach on every boot. You need use a systemd-tmpfiles entry.

Reply:
Ah, of course. great catch!